frontend/packages/knative-plugin/src/components/add/event-sources/ApiServerSection.tsx

```diff
@@ -33,6 +33,7 @@ const ApiServerSection: React.FC = () => {
     Ref: 'Ref',
     Resource: 'Resource',
   };
+  useFormikValidationFix(values);
```

Comment:
@sahil143 Instead of calling `useFormikValidationFix` in each section of the form, i think it would be better to called it once in `EventSourceForm`. Ultimately our goal is to trigger `validateForm` whenever anything inside `values` changes.

Reply:
Agreed. Will update this

Reply:
Adding the hook in EventSourceSection should work.